gensim/models/lda_dispatcher.py

```diff
@@ -181,24 +181,24 @@ def exit(self):
 def main():
     parser = argparse.ArgumentParser(description=__doc__)
     parser.add_argument("--maxsize", help="How many jobs (=chunks of N documents) "
-                                           "to keep 'pre-fetched' in a queue (default: %(default)s)",
-                        type=int, default=MAX_JOBS_QUEUE)
+        "to keep 'pre-fetched' in a queue (default: %(default)s)",
+        type=int, default=MAX_JOBS_QUEUE)
     parser.add_argument("--host", help="Nameserver hostname (default: %(default)s)", default=None)
     parser.add_argument("--port", help="Nameserver port (default: %(default)s)", default=None, type=int)
     parser.add_argument("--no-broadcast", help="Disable broadcast (default: %(default)s)",
-                        action='store_const', default=True, const=False)
+        action='store_const', default=True, const=False)
     parser.add_argument("--hmac", help="Nameserver hmac key (default: %(default)s)", default=None)
     parser.add_argument('-v', '--verbose', help='Verbose flag', action='store_const', dest="loglevel",
-                        const=logging.INFO, default=logging.WARNING)
+        const=logging.INFO, default=logging.WARNING)
     args = parser.parse_args()
 
     logging.basicConfig(format='%(asctime)s : %(levelname)s : %(message)s', level=args.loglevel)
     logger.info("running %s", " ".join(sys.argv))
 
     ns_conf = {"broadcast": args.no_broadcast,
-               "host": args.host,
```

Comment:
Hanging indent has the first line empty (content starts at the first indented line). See the examples in [PEP8](https://www.python.org/dev/peps/pep-0008/#indentation).

(comment applies here and elsewhere)